recipes/simpeg/meta.yaml

```diff
@@ -0,0 +1,95 @@
+package:
+  name: simpeg
+  version: "0.6.2b1"
+
+source:
+  fn: SimPEG-0.6.2b1.tar.gz
+  url: https://pypi.python.org/packages/1b/dd/1ce3fb26388d5a5fec6e4604679721fe3de0cdc0e5242438f64caaf21c7d/SimPEG-0.6.2b1.tar.gz
+  md5: d402a75bc696a5ddd3ffb424b30eda8a
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7
+    - scipy >=0.13
+    - cython
+    - pymatsolver >=0.1.1
+    - ipython
+    - matplotlib
+    - properties[math] >=0.3.1b2
+    - discretize
+
+  run:
+    - python
+    - numpy >=1.7
+    - scipy >=0.13
+    - cython
+    - pymatsolver >=0.1.1
+    - ipython
+    - matplotlib
+    - properties[math] >=0.3.1b2
```

Comment:
You should drop the `[math]` part of `properties`. The conda-forge recipe is built with the math extras requirements already. 

You're going to have to add recipes for `discretize` and `pymatsolver` as I don't think they are available on conda-forge or in the `defaults` channel.

Reply:
right. we need to do that.